opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -324,7 +325,7 @@ CpGrid::scatterGrid(EdgeWeightMethod method,
         }
         else
         {
-            if (useZoltan)
+            if (partitionMethod == 1 or partitionMethod == true)
```

Comment:
This totally confuses me:
If `partitionMethod` is 2 then the first is false, but what is up with the second? Will true be cast to int and the be compared?

Can we plese simply this or is there a reason that I fail to see,

Reply:
Yes, I'll remove that